### PR TITLE
Remove legacy kubernetes-fetching code from e2e-runner

### PR DESCRIPTION
**Pull request by ixdy**
I told you I'd delete all the duplicated code. :)

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/954)

### CODE REVIEW

**Comment by spxtr** [comment]
`+19 −118` :+1:

**Comment by wonderfly** [comment]
How far away is this PR from merging? I had started my #961 before I saw this...

**Comment by ixdy** [comment]
@wonderfly I'll probably merge as soon as I address @spxtr's comments.

File: jenkins/e2e-image/e2e-runner.sh [hunk 1]
```diff
@@ -150,69 +148,6 @@ function call_get_kube() {
 
 ### END FUNCTIONS TO SUPPORT get-kube.sh ###
```
Comment (ixdy) [inline jenkins/e2e-image/e2e-runner.sh]: Done.

**Pull request merged**